.yamato/all-hdrp.yml

```diff
@@ -7,87 +7,121 @@
 
 ---
 
-All_HDRP_2020.2:
-    name: All HDRP CI - 2020.2
+PR_HDRP_2020.2:
+    name: HDRP PR Job - 2020.2
     dependencies:
+      -  path: .yamato/hdrp-win-dx11.yml#HDRP_Win_DX11_Standalone_cache_mono_Linear_2020.2
+         rerun: on-new-revision
+      -  path: .yamato/hdrp-win-dx11.yml#HDRP_Win_DX11_playmode_cache_mono_Linear_2020.2
+         rerun: on-new-revision
+      -  path: .yamato/hdrp-win-dx11.yml#HDRP_Win_DX11_playmode_XR_cache_mono_Linear_2020.2
+         rerun: on-new-revision
       -  path: .yamato/hdrp-win-dx11.yml#HDRP_Win_DX11_editmode_mono_Linear_2020.2
          rerun: on-new-revision
-      -  path: .yamato/hdrp-win-dx11.yml#HDRP_Win_DX11_playmode_mono_Linear_2020.2
+      -  path: .yamato/hdrp-win-dx12.yml#HDRP_Win_DX12_playmode_cache_mono_Linear_2020.2
          rerun: on-new-revision
-      -  path: .yamato/hdrp-win-dx11.yml#HDRP_Win_DX11_playmode_XR_mono_Linear_2020.2
+      -  path: .yamato/hdrp-win-vulkan.yml#HDRP_Win_Vulkan_playmode_cache_mono_Linear_2020.2
          rerun: on-new-revision
-      -  path: .yamato/hdrp-win-dx11.yml#HDRP_Win_DX11_playmode_NonRenderGraph_mono_Linear_2020.2
+      -  path: .yamato/hdrp-osx-metal.yml#HDRP_OSX_Metal_playmode_cache_mono_Linear_2020.2
          rerun: on-new-revision
-      -  path: .yamato/hdrp-win-dx11.yml#HDRP_Win_DX11_Standalone_mono_Linear_2020.2
+      -  path: .yamato/hdrp-linux-vulkan.yml#HDRP_Linux_Vulkan_Standalone_mono_Linear_2020.2
          rerun: on-new-revision
-      -  path: .yamato/hdrp-win-dx12.yml#HDRP_Win_DX12_playmode_mono_Linear_2020.2
+      -  path: .yamato/hdrp-linux-vulkan.yml#HDRP_Linux_Vulkan_playmode_cache_mono_Linear_2020.2
          rerun: on-new-revision
-      -  path: .yamato/hdrp-win-dx12.yml#HDRP_Win_DX12_playmode_XR_mono_Linear_2020.2
+      -  path: .yamato/all-hdrp_dxr.yml#PR_HDRP_DXR_2020.2
          rerun: on-new-revision
-      -  path: .yamato/hdrp-win-dx12.yml#HDRP_Win_DX12_Standalone_mono_Linear_2020.2
+      -  path: .yamato/all-hdrp_hybrid.yml#PR_HDRP_Hybrid_2020.2
          rerun: on-new-revision
-      -  path: .yamato/hdrp-win-vulkan.yml#HDRP_Win_Vulkan_playmode_mono_Linear_2020.2
+      -  path: .yamato/all-vfx_hdrp.yml#PR_VFX_HDRP_2020.2
          rerun: on-new-revision
-      -  path: .yamato/hdrp-win-vulkan.yml#HDRP_Win_Vulkan_playmode_XR_mono_Linear_2020.2
+    triggers:
+        expression: pull_request.target eq "master" AND NOT pull_request.draft AND pull_request.push.changes.all match ["com.unity.render-pipelines.high-definition/**/*", "com.unity.render-pipelines.high-definition-config/**/*",  "TestProjects/HDRP_Tests/**/*"]
```

Comment:
Related to prev. comment: it would also prevent docs from triggering the abv https://github.com/Unity-Technologies/Graphics/pull/3254